components/gitpod-protocol/data/gitpod-schema.json

```diff
@@ -136,13 +136,38 @@
             },
             "additionalProperties": false
         },
+        "additionalRepositories": {
```

Comment:
> How about childRepositories and parentRepository?

I had that and changed it because of feedback I got from others in the vain of "hey there are no primary and secondary repositories, it's just the one that contains the configuration"

Reply:
Just wanted to leave an upvote for `parent`/`child`: it immediately conveys the meaning. And reflects the (default) file system layout.

Reply:
The default file system layout is not hierarchical, but flat (i.e. siblings)

Reply:
> The default file system layout is not hierarchical, but flat (i.e. siblings)

I thought I saw a "/workspace/parent-repo/sub-repo" folder structure in the loom video - but now those are siblings. :+1:

Reply:
Orthogonally to the `parent`/`child` vs `main`/`additional` discussion (which I no longer have strong feelings about), my main gripe with the current naming is actually with the word `Configuration`:
- https://github.com/svenefftinge/multi-repo-main is _not_ a configuration, but a repository. Calling it a configuration leads to confusion.
- Trying `mainConfiguration: https://github.com/svenefftinge/multi-repo-main/blob/main/.gitpod.yml` will _not_ work as the naming suggests.
- Calling one property `...Configuration` and the other `...Repositories` is also confusing:
  - They don't appear to use the same type (one wants repository URLs, the other wants... a configuration? 🤔)
  - They don't look related at all (even though you're mostly supposed to use them together to get multi-repo support)

What do you think of alternatives like `mainRepository`, or `configuringRepository`, or `configurationRepository`?

Reply:
After this we might likely introduce multiple `.gitpod.yml` per repo (i.e. the other side) to support opening workspaces on subparts of a large repo (big mono-repos or example projects). I have a use case in mind, where I would have multiple `.gitpod.yml` in one repo where some delegate to the same `mainConfiguration` (sitting in the root of the same repo) but would overlay properties such as `workspaceLocation` to open the IDE on a subpart of the repo's contents.